app/javascript/components/visualization/ScatterPlot.jsx

```diff
@@ -232,7 +373,7 @@ function RawScatterPlot({
     })
   }, [cluster, annotation.name, subsample, consensus, genes.join(','), isAnnotatedScatter])
 
-  // Handles custom scatter legend updates
+  // // Handles custom scatter legend updates
```

Comment:
double commented out might want to remove one set of `//`

Reply:
Done in 505c492.